blog/2019-09-10-debezium-0-10-0-cr1-released.adoc

```diff
@@ -0,0 +1,95 @@
+there's still several use cases working with such values, e.g. think of potentially large half-life times when modelling nuclear decay processes.
+So we did a larger refactoring of the code dealing with temporal values, and we are happy to report that these issues have been fixed now.
+
+Other bugfixes were done in regards to dealing with "no-op" events in MongoDB
+(https://issues.jboss.org/browse/DBZ-1464[DBZ-1464]) and the recently added propagation of `source` fields to outgoing message when using the link:/documentation/reference/0.10/configuration/event-flattening.html[new record state extraction SMT] (https://issues.jboss.org/browse/DBZ-1448[DBZ-1448]).
+
+== Cassandra Connector
+
+After the link:/blog/2019/08/20/debezium-0-10-0-beta4-released/[initial release] of the Debezium connector for Cassandra,
+work has begun to further align it with the other Debezium connectors
+(unlike the relational connectors and the one for MongoDB, the Cassandra connector currently is not based on Apache Kafka Connect,
+but runs as a stand-alone process).
+The first outcome of this is that its configured via a properties file now
+(similar to using Kafka Connect in standalone mode) and not via a YAML file.
+(https://issues.jboss.org/browse/DBZ-1406[DBZ-1406]).
+
+The next step will be to make the aspect of message serialization configurable:
+while currently only Avro is supported by the connector,
+it will eventually support the notion of pluggable converters,
+allowing you to use the JSON, Avro and any other converters you may already know from Kafka Connect.
+
+== Reworked Website and Documentation
+
+When reading this blog post, it's hard to miss:
+the Debezium website has link:/blog/2019/09/05/website-documentation-overhaul/[received a facelift].
+
+Information for the current stable and development releases (0.9 and 0.10 at this time) is much easier to find now.
+Also the documentation has been re-organized and is published in version-specific approach now,
+i.e. you can now obtain the specific documentation applying for a particular release.
+
+The blog section of the website has been reworked, too:
+the main page shows an introductory snippet for the most recent posts,
+whereas a number of "featured" blog posts is listed on the left.
+These are typically earlier blog posts which explore advanced topics such as the link:/blog/2019/02/19/reliable-microservices-data-exchange-with-the-outbox-pattern/[outbox pattern] in depth and which we wanted to make easier to discover and consume.
+We hope you like the new website and documentation structure and it helps you to find all the information you're looking for better than before.
+If you run into any issues (formatting glitches, broken links etc.), please let us know.
+
+Refer to the link:/docs/releases/#release-0-10-0-cr1[0.10.0.CR1 release notes] for the complete list of addressed issues and the upgrading procedure.
```

Comment:
@Naros, I'm unsure which link I should add here; could you help out?

Reply:
`/releases/0.10/release-notes##release-0-10-0-cr1`

Reply:
Thx, adjusted.